src/ProjectTemplates/Web.ProjectTemplates/content/RazorPagesWeb-CSharp/.template.config/template.json

```diff
@@ -339,6 +386,19 @@
   ],
   "defaultName": "WebApplication",
   "postActions": [
+    {
+      "condition": "(CallsMicrosoftGraph)",
+      "actionId": "B17581D1-C5C9-4489-8F0A-004BE667B814",
+       "description": "Adds Microsoft Graph package.",
+      "manualInstructions": [
+      ],
+      "args": {
+        "referenceType": "package",
+        "reference": "Microsoft.Graph",
+        "version": "3.8.0"
```

Comment:
Why is this added as a postAction instead of a conditional package reference in the csproj? @jmprieur?

Reply:
The template engine did not seem to apply the condition to the csproj (or I have not found out how to do it).
Do you have an example of where this is done, @JunTaoLuo  ?